src/server/utils/auth.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+const { SECRET } = require('../config');
+
+const saltRounds = 10;
+
+const hashedPassword = async(password) => {
+    return bcrypt.hashSync(password, saltRounds);
+}
+
+const checkPassword = async(password, foundedPassword) => {
+    try {
+        return await bcrypt.compare(password, foundedPassword);
+    } catch (error) {
+        return error;
+    }
+}
+
+const createToken = async(payload) => {
+    return jwt.sign(payload, SECRET);
+}
+
+const removeKeys = (user, ...keys) => {
```

Comment:
I'd rename this `user` parameter to something more generic. You could use this function with any object, not just users!